lib/api/ban-id/helpers.js

```diff
@@ -0,0 +1,25 @@
+import {v4 as uuidv4} from 'uuid'
+import {idsInDataBase} from './models.js'
+
+export const getUuids = length => {
+  const ids = new Set()
+  while (ids.size !== length) {
+    ids.add(uuidv4())
+  }
+
+  return [...ids]
+}
+
+export const uncollidUuids = async ids => {
+  const bddIds = await idsInDataBase(ids)
+  const filteredIds = ids.filter(id => !bddIds.includes(id))
+  if (filteredIds.length !== ids.length) {
+    while (filteredIds.length !== ids.length) {
```

Comment:
```suggestion
    filteredIds.push(...getUuids(filteredIds.length))
```
on pourrai utiliser getUuids car ici collision not safe alors que la getUuids ok

Reply:
Bien vu, mais ce ne serait pas plutôt :
```javascript
filteredIds.push(...getUuids(ids.length - filteredIds.length))
```
Sinon, la taille finale de `filteredIds` risque d'être plus grande que la taille attendu.